docs/troubleshooting/management/ts-management.asciidoc

```diff
@@ -78,3 +78,62 @@ image::images/transforms-start.png[Transforms page with Start option selected]
 . On the confirmation message that displays, click *Start* to restart the transform.
 . The transform’s status changes to `started`. If it doesn't change, refresh the page.
 ====
+
+[discrete]
+[[ts-agent-connection]]
+.{agent} and Endpoint connection issues
+[%collapsible]
+====
+
+After {agent} installs Endpoint, Endpoint connects to {agent} over a local relay connection to report its health status and receive policy updates and response action requests. If that connection cannot be established, the {elastic-defend} integration will cause {agent} to be in an `Unhealthy` status, and Endpoint won't operate properly.
+
+[float]
+==== Identify if the issue is happening
+
+You can identify if this issue is happening in the following ways:
+
+* Run {agent}'s status command:
+** `sudo /opt/Elastic/Agent/elastic-agent status` (Linux)
+** `sudo /Library/Elastic/Agent/elastic-agent status` (macOS)
+** `c:\Program Files\Elastic\Agent\elastic-agent.exe status` (Windows)
+
+If the status result for `endpoint-security` says that Endpoint has missed check-ins or `localhost:6788` cannot be bound to, it might indicate this problem is occurring.
+
+* If the problem starts happening right after installing Endpoint, check the value of `fleet.agent.id` in the following file:
+** `/opt/Elastic/Endpoint/elastic-endpoint.yaml` (Linux)
+** `/Library/Elastic/Endpoint/elastic-endpoint.yaml` (macOS)
+** `c:\Program Files\Elastic\Endpoint\elastic-endpoint.yaml` (Windows)
+
+If the value of `fleet.agent.id` is `00000000-0000-0000-0000-000000000000`, this indicates this problem is occurring.
+
+NOTE: If this problem starts happening after Endpoint has already been installed and working properly, then this value will have changed even though the problem is happening.
+
+[float]
+==== Examine Endpoint logs
+
+If you've confirmed that the issue is happening, you can look at Endpoint log messages to identify the cause:
+
+* `Failed to find connection to validate. Is {agent} listening on 127.0.0.1:6788?` or `Failed to validate connection. Is {agent} running as root/admin?` means that Endpoint is not able to create an initial connection to {agent} over port `6788`.
```

Comment:
This didn't render like you wanted. Since it's a literal log message, I think it makes sense to leave `{agent}` as `Agent`.

<img width="818" alt="image" src="https://github.com/elastic/security-docs/assets/56368752/05c0e309-7e54-4a8d-80d8-9a2fe847557f">